File: C# ASP.NET/FilmsAndGenres/FilmsAndGenres/Models/Film.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FilmsAndGenres.Models
{
    public class Film
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Year { get; set; }
        public int GenreId { get; set; }
    }

    public class FilmForShow
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Year { get; set; }
        public string Genre { get; set; }
    }
}

File: Android/Android_8_4_castomWidgetToListView/app/src/main/java/com/example/android_8_4_castomwidgettolistview/Film.java
package com.example.android_8_4_castomwidgettolistview;

public class Film
{
    public  String title;
    public  String genre;
    public  int year;
    public  Film(String title, String genre, int year)
    {
        this.title = title;
        this.genre = genre;
        this.year = year;
    }
    @Override
    public String toString()
    {
        return  "Title: "+this.title+", Genre: "+this.genre+", Year: "+this.year;
    }

}
